src/gt4py/analysis/passes.py

```diff
@@ -255,6 +255,12 @@ def visit_Assign(self, node: gt_ir.Assign):
 
             return result
 
+        def visit_NativeFuncCall(self, node: gt_ir.NativeFuncCall):
+            extents = []
+            for arg in node.args:
+                extents.extend(self.visit(arg))
```

Comment:
Since I had earlier suggested a wrong way to code this in one line, here it is how it could be actually done:
```python
extents = [extent for arg in node.args for extent in self.visit(arg)]
```
Just as a curiosity, it does not really improve the current code.

Reply:
Ah ha, that is a nice trick!